### PR TITLE
Feature/submit shoe

### PR DESCRIPTION
**Pull request by jeremiahblackol**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

### Files modified:
- [ ] App.css

### CODE REVIEW

**Comment by Aeickelman40** [comment]
Nice, thanks for taking care of those styling changes. Looks a lot better!

File: src/App/App.css [hunk 1]
```diff
@@ -15,30 +16,71 @@
...
+  line-height: 1em;
+  color: rgba(0, 128, 0, 0.555);
+  text-shadow: 7px -7px rgb(43, 6, 6);
+
+  font-size: 8.7em;
+  line-height: 1em;
+  margin-bottom: -.2em;
+}
+
+.header h2 {
```
Comment (jdvolk) [inline src/App/App.css:52]: Looks good!

File: src/App/App.css [hunk 2]
```diff
@@ -47,23 +89,58 @@ background-color: #a5a58d;
...
+
+.colorInput {
+  font-size: 51px;
+  width: 3.9em;
+  /* height: 51px; */
 }
 
 .submitButton {
-  background: #ddbea9;
+  background: #5f0101;
```
Comment (jdvolk) [inline src/App/App.css:126]: Like the color change

File: src/SubmitShoeForm/SubmitShoeForm.js [hunk 3]
```diff
@@ -79,7 +81,7 @@ class SubmitShoeForm extends Component {
                 type='text'
                 placeholder='Model' required>
             </input>
-            <button type="submit" onClick={this.addId}>
+            <button type="submit" onClick={this.addId} className="submitButton" >
```
Comment (jdvolk) [inline src/SubmitShoeForm/SubmitShoeForm.js:84]: this works beautifully

**Pull request merged**